lib/src/window_manager.dart

```diff
@@ -177,11 +190,41 @@ class WindowManager {
 
   /// Returns `bool` - Whether app is active
   ///
-  /// @platforms macos
+  /// @platforms macos,windows
   Future<bool> isActive() async {
     return await _channel.invokeMethod('isActive');
   }
 
+  /// Returns `bool` - Whether app is hidden on Blur Event
+  ///
+  /// @platforms windows
+  Future<bool> isHiddenOnBlur() async {
```

Comment:
How does this difer from `isHideWindowOnDeactivate`?

Reply:
refer to the above point for a detail explanation.